src/Plugin/Patches.php

```diff
@@ -443,6 +443,14 @@ protected function getAndApplyPatch(RemoteFilesystem $downloader, $install_path,
             // differences between how patch works on windows and unix.
             $patch_options = '--no-backup-if-mismatch';
             if (PHP_OS_FAMILY == 'BSD') {
```

Comment:
This won't work reliably because PHP_OS_FAMILY reflect the _build target_ for the running PHP version, not the current hosting environment. In my case on MacOS Ventura, for whatever reason, PHP_OS_FAMILY is Linux.

Reply:
I think what you're saying is the PHP documentation is wrong: When it says "The operating system family PHP was built for" what they actually mean is "The operating system family PHP was built on"?  If so, uhg..

So maybe looking at patch --version is correct after all, although all the logic needed there might not be pretty.

Reply:
Yeah the only semi-reliable way to detect runtime OS is [php_uname](https://www.php.net/manual/en/function.php-uname.php), but even that will fall back to displaying the _build_ OS in some situations. I found the [phposinfo](https://github.com/loophp/phposinfo) library to do a pretty good job of compensating for this by using a combination of php_uname, PHP_OS_INFO, and shell commands to verify the OS version.

It's obviously better to avoid any conditional logic based on the OS in the first place.